### PR TITLE
documentation/basic-ad-update

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** #35 and #32 

**This PR will result in the following new package version:** `v0.7.1`

This will not be a breaking change as this only updates documentation.

## Remaining PR components
Please see the accompanying [source PR](https://github.com/fivetran/dbt_facebook_ads_source/pull/31) which contains all the same information.

### CODE REVIEW

**Review by fivetran-catfritz — APPROVED** [review]
lgtm!

**Review by fivetran-catfritz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# dbt_facebook_ads v0.7.1
+
+[PR #36](https://github.com/fivetran/dbt_facebook_ads/pull/36) includes the following updates:
+## Documentation Updates
+- The prerequisite steps for generating the `basic_ad` pre-built report have been modified to reflect the current state of the Facebook Ads connector.
```
Comment (fivetran-reneeli) [inline CHANGELOG.md:5]: same comment as in source

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md]: Good call. Just updated

**Review by fivetran-reneeli — APPROVED** [review]
lgtm!

**Pull request merged**